src/test/java/ch/uzh/ifi/hase/soprafs22/service/UserServiceIntegrationTest.java

```diff
@@ -81,8 +84,10 @@ void createUser_duplicateUsername_throwsException() {
     testUser.setStatus(UserStatus.ONLINE);
     testUser.setToken("a");
     testUser.setPassword("1234");
+    testUser.setBirthday(new Date());
```

Comment:
Why do you set the birthday as today (would make more sense to have him be older than 18)